### PR TITLE
KAZOO-5028 disable conference call-control

### PR DESCRIPTION
**Pull request by jamhed**
- disable freeswitch built-in defaults for conference call-controls

### CODE REVIEW

File: applications/ecallmgr/src/ecallmgr_fs_config.erl [hunk 1]
```diff
@@ -449,6 +449,7 @@ maybe_fetch_conference_profile(Node, Id, Profile) ->
                       {'ok', Resp} = ecallmgr_fs_xml:not_found(),
                       Resp
               end,
+    lager:error("CONFIG:~p", [XmlResp]),
```
Comment (fenollp) [inline applications/ecallmgr/src/ecallmgr_fs_config.erl]: Missed this?

**Comment by lazedo** [comment]
@jamhed why do you want to disable conference call-controls ?

**Comment by jamhed** [comment]
@lazedo, because call control is handled by kazoo, not mod_conference.

**Comment by lazedo** [comment]
so how do i mute my-self from deskphone ?

**Comment by jamhed** [comment]
@lazedo by sending dtmf *1

**Comment by lazedo** [comment]
merging manually

**Pull request closed without merging**